src/components/Text/src/Text.vue

```diff
@@ -23,48 +22,52 @@ export default {
 
 	props: {
 		/**
-		 * which HTML element to wrap the text with
+		 * HTML Element wrapper
 		 */
 		element: {
 			type: String,
 			default: 'p',
 			validator: (element) => ['p', 'span'].includes(element),
 		},
 		/**
-		 * size of text
+		 * Size of text
+		 * @values 7, 6, 5, 4, 3, 2, 1, 0, -1, -2
```

Comment:
this should be 1, 0, -1, -2

look at MIN_SIZE and MAX_SIZE consts defined above.